core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -233,6 +233,14 @@ private[spark] class ExecutorAllocationManager(
   }
 
   /**
+   * Change the value of numExecutorsTarget.
+   */
+  def reSetNumExecutorsTarget(): Unit = {
```

Comment:
I would just call this `reset()` since it needs to do more than just setting the target